### PR TITLE
SeleneStation: Moving the wrench ,in the execution room closet, outside

### PR DESCRIPTION
**Pull request by EIB0N**
I need mapping jobs, I'm going hungry

## About The Pull Request

Moved the wrench outside

## Why It's Good For The Game

Officers can now wrench the canisters inside

## Changelog
:cl:

add: Wrench in execution room now accessible to officers

/:cl:

### CODE REVIEW

**Comment by JohnFulpWillard** [comment]
it was missing a wrench, for wrenching the canisters in.

**Comment by EIB0N** [comment]
Bruh it's in the locker in the execution room

**Comment by EIB0N** [comment]
the closet do be high clearance though, lets put it outside

**Comment by SgtHunk** [comment]
> I need mapping jobs, I'm going hungry

Make all of SeleneStation into directional wall mounts coward

**Comment by EIB0N** [comment]
> > I need mapping jobs, I'm going hungry
> 
> Make all of SeleneStation into directional wall mounts coward

Don't tempt me

**Comment by SgtHunk** [comment]
> Don't tempt me

no please this is an actual request

**Comment by Helianthus00** [comment]
why did you remove the bodybags?

**Comment by JohnFulpWillard** [comment]
> 
> 
> why did you remove the bodybags?

:trollface:

File: _maps/map_files/SeleneStation/SeleneStation.dmm [hunk 1]
```diff
@@ -24603,10 +24604,8 @@
 	pixel_x = -3;
 	pixel_y = 1
 	},
-/obj/item/storage/box/bodybags,
```
Comment (Helianthus00) [inline _maps/map_files/SeleneStation/SeleneStation.dmm:24606]: Why remove these bodybags?

Comment (EIB0N) [reply _maps/map_files/SeleneStation/SeleneStation.dmm:24606]: I don't remember every deleting that wut

**Pull request merged**